docs/asciidoc/modules/ROOT/pages/import/web-apis.adoc

```diff
@@ -69,7 +69,7 @@ In order to use https://cloud.google.com/storage/[Google Cloud Storage], you nee
 * google-auth-library-oauth2-http-0.17.1.jar
 * google-cloud-core-1.90.0.jar
 * google-cloud-core-http-1.90.0.jar
-* google-cloud-storage-1.90.0.jar
+* google-cloud-storage-2.22.5.jar
```

Comment:
I'm not sure if we want to update this one, given that the uber-jar described below is old and does not contain this version.

Reply:
Hmm I would also assume the other dependencies are wrong as well, e.g google-cloud-core-http-1.90.0.jar most likely is also 2.22.5 if it is a dependency of cloud storage

Reply:
Yeah, looking at the compile dependencies of google-cloud-storage, these are all different, I would not add this change in this PR but make a card to fix this up along with the uber jar maybe? https://mvnrepository.com/artifact/com.google.cloud/google-cloud-storage/2.22.5

Reply:
Sounds good!